src/comps/Header.tsx

```diff
@@ -63,6 +62,10 @@ const Header = ({ config }: { config: BridgeConfig }) => {
       </>
     );
   };
+
+  if (!config) {
+    return null;
+  }
```

Comment:
this will never happen config will always be passed

Reply:
Remvoed